test/cypress/integration/remote_registry.js

```diff
@@ -49,8 +95,24 @@ describe('Remote Registry Tests', () => {
 
   it('admin can add new remote registry', () => {
     cy.menuGo('Execution Enviroments > Remote Registries');
-    cy.addRemoteRegistry('New remote registry1', 'some url1');
-    cy.addRemoteRegistry('New remote registry2', 'some url2');
+    addData('New remote registry1', 'some url1', {
+      username: 'some username1',
+      password: 'some password1',
+      proxy_url: 'some proxy_url1',
+      proxy_username: 'some proxy_username1',
+      proxy_password: 'some proxy_password1',
+      download_concurrency: 5,
+      rate_limit: 5,
+    });
```

Comment:
```suggestion
    addData('New remote registry1', 'some url1');
```

(so that we're still testing the simple version of the form as well)